src/Ledger/Transaction.lagda

```diff
@@ -37,17 +37,27 @@ record TransactionStructure : Type₁ where
 
 Transactions are defined in Figure~\ref{fig:defs:transactions}. A
 transaction is made up of a transaction body, a collection of
-witnesses and some optional auxiliary data. Some key ingredients in
-the transaction body are:
-
+witnesses and some optional auxiliary data.
+\begin{NoConway}
+Some key ingredients in the transaction body are:
 \begin{itemize}
   \item A set \AgdaField{txins} of transaction inputs, each of which identifies an output from a previous transaction.
     A transaction input consists of a transaction id and an index to uniquely identify the output.
   \item An indexed collection \AgdaField{txouts} of transaction outputs.
     The \TxOut type is an address paired with a coin value.
   \item A transaction fee. This value will be added to the fee pot.
-  \item The size and the hash of the serialized form of the transaction that was included in the block.
+  \item The size \AgdaField{txsize} and the hash \AgdaField{txId} of the serialized form of the transaction that was included in the block.
+\end{itemize}
+\end{NoConway}
+\begin{Conway}
```

Comment:
@WhatisRT do lines 52--59 look okay to you? If not, can you suggest better descriptions for these new fields of TxBody?